uvloop/const.py

```diff
@@ -0,0 +1,2 @@
+FS_EVENT_CHANGE = 2
+FS_EVENT_RENAME = 1
```

Comment:
I would suggest make this an enum to be more friendly to Python users.

Reply:
now an enum